### PR TITLE
Feature/jjdd8 ism 3

### PR DESCRIPTION
**Pull request by GregChyla**
Deleted old version of tree menu to simplify it into switch-based menu.

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/menu/App/Menu.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        System.out.println("Witaj w kalendarzu wydarzeń kulturalnych");
+        System.out.println("Najbliższe nadchodzące wydarzenia:");
+        //TODO: POkaz najbliższe wydarzenia
+
+        System.out.println("Wybierz menu:");
+        System.out.println("1. Pokaż wszystkie wydarzenia");
+        System.out.println("2. Pokaż wydarzenia wg organizatora");
+        System.out.println("3. Znajdź wydarzenie wg daty");
+        System.out.println("");
+        System.out.println("Wpisz liczbę");
```
Comment (ChrisMcCat) [inline src/main/java/menu/App/Menu.java]: mieliśmy używać "stdout.info" zamiast sout do druku na konsoli dla sonara

File: src/main/resources/log4j2.xml [hunk 2]
```diff
@@ -1,13 +0,0 @@
...
-    <Console name="Console">
-      <PatternLayout pattern="%msg"/>
-    </Console>
-  </Appenders>
-  <Loggers>
-    <Logger name="CONSOLE_OUT" level="info" additivity="false">
-      <AppenderRef ref="Console"/>
-    </Logger>
-  </Loggers>
-</Configuration>
```
Comment (ChrisMcCat) [inline src/main/resources/log4j2.xml]: czemu usuwasz plik dostarczony od Rafała dla loggera ?

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/menu/App/Menu.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+        switch (choice) {
+            case 1:
+                stdout.info("Wybór menu 1\n");
+                new Submenu1().doSomething();
+                break;
+            case 2:
+                //TODO metoda2
+                stdout.info("Wybór menu 2\n");
+                new Submenu2().doSomething();
```
Comment (rafalmisiak) [inline src/main/java/menu/App/Menu.java]: Te nazwy klas to zakładam że WIP

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request closed without merging**